src/macros/xed_enum.rs

```diff
@@ -35,13 +36,11 @@ macro_rules! xed_enum {
                     );
                 )*
             }
-        }
 
-        impl $name {
             #[doc = concat!(
                 "Create a `", stringify!($name), "` from the underlying enum value."
             )]
-            pub const fn from_raw(value: u32) -> Option<Self> {
+            pub(crate) const fn from_raw(value: u32) -> Option<Self> {
```

Comment:
That this is `const` is actually rather important. It means that users of `xed` can create their own constants for new variants introduced by `xed-sys` before we get around to adding them here.